src/components/header/header.tsx

```diff
@@ -20,57 +20,76 @@ const Header = () => {
       position="sticky"
       id="header"
       top={0}
-      py={{ sm: "xs", md: "s" }}
-      px="s"
       zIndex={999}
       bgColor="ui.white"
+      pt="xs"
     >
       <Box
-        alignItems="center"
         maxWidth="1280px"
-        padding={{ md: "0 15px" }}
         mx="auto"
+        padding="0 16px"
+        sx={{
+          "@media screen and (min-width: 1024px)": {
```

Comment:
Can you define the values as variables to not have magic numbers and to more easily see that, for example, 1024px is "large tablet"? It _might_ make it easier to update these queries in the future once they are set in the DS as well.